public/Projects/AmanTyagi/faq-accordion/Code files/index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```

Comment:
add some things like that
<meta name="author" content=<your_username> />
<meta name="description" content=<somethings_like_that_bla_bla> />